src/utils.js

```diff
@@ -20,14 +18,18 @@ const GIT_CONFIG_COMMAND = "git config --global";
 
 export default {
 	readFile( path ) {
-		try {
-			return fs.readFileSync( path, "utf-8" );
-		} catch ( exception ) {
-			return "";
+		if ( path ) {
+			try {
+				return fs.readFileSync( path, "utf-8" );
+			} catch ( exception ) {
+				return null;
+			}
 		}
+
+		return null;
 	},
 	readJSONFile( path ) {
-		const content = this.readFile( path );
+		const content = this.readFile( path ) || "{}";
```

Comment:
this seems similar to my other comment. was this added because of unit tests? if so, can the tests be fixed so this isn't needed?